docs/protocol/Consensus.md

```diff
@@ -0,0 +1,25 @@
+# Oraclization (message)
```

Comment:
This is an ugly as hell document which is out of scope for this PR. I just moved it out of a document I did correct under this PR.